### ISSUE TITLE
TES5EDIT through Mod Organizer reports non-existent ITMs in USLEEP

### ISSUE THREAD

**Issue by Elaura1970** [TanninOne/modorganizer#946]
Skyrim Legendary (Steam)
Using MO v 1.3.11
TES5EDIT v 3.1.3
USLEEP v 3.0.5

Running TES5Edit through MO reports 227 (erroneous) ITMs in USLEEP. Running TES5Edit directly from the executable reports no ITMs.

Discussion with the USLEEP folks: http://forums.bethsoft.com/topic/1538020-relz-unofficial-skyrim-legendary-edition-patch-usleep-1/page-7#entry25213158

**Comment by GSDFan** [comment]
I am not seeing this on my system. Is it possible you have another copy of usleep in the overwirte folder or another folder. Try making a new profile with no other mods enabled and see if this makes any difference.

**Comment by Elaura1970** [comment]
No, as it turned out, it was a result of having MO manage archives.  
When I unchecked that, the ITMs weren't found.  I can't reproduce the 
error in a fresh profile, but my modded profile is consistent.

Just so you know, I've done some extensive troubleshooting on this 
already. The first thing I did was run TES5Edit outside of MO with 
freshly cleaned masters and no ITMs were reported.  I verified cache 
files through Steam as well. Nothing, from Stem on down is installed in 
any system drive or directory.

I also moved TES5Edit out of the Skyrim folder, then I moved MO out of 
the Skyrim folder.  Both still ran perfectly, but reported the ITMs when 
I ran Xedit through MO.

I also used a freshly d/l version of Xedit and re-unzipped USLEEP, 
making sure to remove all existing installations completely.


On a fresh install with nothing but USLEEP and the cleaned masters 
checked in *both* the left and right panes, no ITMs are reported whether 
the manage BSAs box is checked or not. It is the same when adding in the 
Hi-Res bsas and the Unofficial Hi-res Patch bsa.

Past that, I haven't tested. I will re-install STEP 2.9.2 from scratch, 
to a new profile and try again. It appears load order, either in the 
left or right pane has some influence on the number of ITMs reported, as 
my current load order is now reporting only 119.

Thanks for getting back to me on this.

- Elaura
